Sources/SwiftDocC/Infrastructure/External Data/ExternalMetadata.swift

```diff
@@ -41,6 +42,18 @@ public struct VersionTriplet: Equatable, Comparable {
         self.minor = minor
         self.patch = patch
     }
+
+    public init(_ semanticVersion: SemanticVersion) {
+        self.major = semanticVersion.major
+        self.minor = semanticVersion.minor
+        self.patch = semanticVersion.patch
+    }
```

Comment:
What's the conceptual difference between a Version Triplet and a SemanticVersion? Do we need both?